k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```diff
@@ -101,7 +101,7 @@ spec:
                       - key: JENKINS_CONFIG_REPO
                         value: sds-jenkins-config
                       - key: BUSINESS_AREA_TAG
-                        value: Cross-Cutting
+                        value: CFT
```

Comment:
err you want cross-cutting here

Reply:
Previous typo which i forgot to restore.  Not supposed to be part of this changed